src/m_config.c

```diff
@@ -2201,11 +2203,11 @@ void M_SetConfigDir(const char *dir)
     if (strcmp(configdir, "") != 0)
     {
         printf("Using %s for configuration and saves\n", configdir);
-    }
+        // Make the directory if it doesn't already exist:
```

Comment:
What's the purpose of this change?

Reply:
mkdir with "" as path  crashes on Switch atm. This seems like it might possibly be UB elsewhere too.